### PR TITLE
Export Schema

### PR DESCRIPTION
**Pull request by rchinn-segment**
### Proposed changes
Adding Export Schema feature to the documentation. 

### Merge timing

Once approved

### CODE REVIEW

**Comment by mgutten** [comment]
Looks great! Awesome job @rchinn-segment

**Review by mgutten — APPROVED** [review]

File: src/connections/destination-data-control.md [hunk 1]
```diff
@@ -38,3 +38,26 @@ If you no longer want to track an event, you can either remove it from your code
...
+- Allowed Count
+- Blocked Count
+- Total Count
+- Planned (available for Protocols customers with a connected Tracking Plan)
+  - values are "planned" or "unplanned"
+
+Export Schema is not available for Identify or Group events.
+
+> note ""
+> Export Schema does not include any actual values (for example, personal data) for the events, properties, and traits you are tracking for a specific source. Properties are not included in the CSV.
```
Comment (stayseesong) [inline src/connections/destination-data-control.md:63]:
```suggestion
> Export Schema doesn't include any actual values (for example, personal data) for the events, properties, and traits you are tracking for a specific source. Properties aren't included in the CSV.
```

File: src/connections/destination-data-control.md [hunk 2]
```diff
@@ -38,3 +38,26 @@ If you no longer want to track an event, you can either remove it from your code
 ![](/docs/protocols/images/event-filters.png)
 
 Once you block an event in Segment, we'll stop forwarding it to all of your Destinations, including your warehouses. You can remove it from your code at your leisure. In addition to blocking track calls, Business plan customers can block all Page and Screen calls, as well as Identify traits and Group properties. 
+
+## Export your Source Schema
+
+Segment allows you to download your Source Schema as a CSV file, maximizing portability and access to event data. You can download a copy of your schema by visiting the source schema page.
```
Comment (stayseesong) [inline src/connections/destination-data-control.md:44]:
```suggestion
Segment allows you to download your Source Schema as a CSV file, maximizing portability and access to event data. You can download a copy of your schema by visiting the Source Schema page.
```

File: src/connections/destination-data-control.md [hunk 3]
```diff
@@ -38,3 +38,26 @@ If you no longer want to track an event, you can either remove it from your code
...
+
+![](images/export-source-schema.png)
+
+CSV files are generated based on the current view of your Source Schema. Any search parameters or filters applied to the current Source Schema view are also applied to the CSV.
```
Comment (stayseesong) [inline src/connections/destination-data-control.md:48]:
```suggestion
CSV files generate based on the current view of your Source Schema. Any search parameters or filters you apply to the current Source Schema view also apply to the CSV.
```

File: src/connections/destination-data-control.md [hunk 4]
```diff
@@ -38,3 +38,26 @@ If you no longer want to track an event, you can either remove it from your code
 ![](/docs/protocols/images/event-filters.png)
 
 Once you block an event in Segment, we'll stop forwarding it to all of your Destinations, including your warehouses. You can remove it from your code at your leisure. In addition to blocking track calls, Business plan customers can block all Page and Screen calls, as well as Identify traits and Group properties.
```
Comment (stayseesong) [inline src/connections/destination-data-control.md:40]:
```suggestion
Once you block an event in Segment, Segment stops forwarding it to all of your Destinations, including your warehouses. You can remove it from your code at your leisure. In addition to blocking track calls, Business plan customers can block all Page and Screen calls, as well as Identify traits and Group properties. 
```

File: src/connections/destination-data-control.md [hunk 5]
```diff
@@ -38,3 +38,26 @@ If you no longer want to track an event, you can either remove it from your code
...
+Segment allows you to download your Source Schema as a CSV file, maximizing portability and access to event data. You can download a copy of your schema by visiting the source schema page.
+
+![](images/export-source-schema.png)
+
+CSV files are generated based on the current view of your Source Schema. Any search parameters or filters applied to the current Source Schema view are also applied to the CSV.
+
+You can download the following Track events:
+- Event Name
+- Last Seen (UTC)
+  - if greater than 30 days, the value is "more than 30 days ago"
```
Comment (stayseesong) [inline src/connections/destination-data-control.md:53]:
```suggestion
  - If greater than 30 days, the value is "more than 30 days ago"
```

Comment (stayseesong) [reply src/connections/destination-data-control.md:53]: Is "more" be capitalized in the UI?

File: src/connections/destination-data-control.md [hunk 6]
```diff
@@ -38,3 +38,26 @@ If you no longer want to track an event, you can either remove it from your code
...
+- Allowed Count
+- Blocked Count
+- Total Count
+- Planned (available for Protocols customers with a connected Tracking Plan)
+  - values are "planned" or "unplanned"
```
Comment (stayseesong) [inline src/connections/destination-data-control.md:58]:
```suggestion
  - Values are "planned" or "unplanned"
```

Comment (stayseesong) [reply src/connections/destination-data-control.md:58]: should "planned" and "unplanned" be capitalized?

File: src/connections/destination-data-control.md (hunk 5 above)
Comment (rchinn-segment) [reply src/connections/destination-data-control.md:53]: "more" is lowercase in the UI/CSV file. Great question.

File: src/connections/destination-data-control.md (hunk 6 above)
Comment (rchinn-segment) [reply src/connections/destination-data-control.md:58]: "planned" and "unplanned" are also printed as lowercase in the CSV.

**Review by stayseesong — APPROVED** [review]

**Pull request merged**